test/jdk/jdk/incubator/vector/unit_tests.template

```diff
@@ -0,0 +1,8647 @@
+/*
```

Comment:
This file should not be part of the commit, nor should `perf_scalar_tests.template` and `perf_tests.template`